packages/esm-env/dev-fallback.js

```diff
@@ -1,13 +1,6 @@
-if (!import.meta.env) {
-	// changes to import.meta only apply to this file
-	// we have to do this to be able to directly access import.meta.env.DEV
-	// bunlder defines such as Vite will only replace import.meta.env.DEV and not import.meta.env?.DEV
-	// we also can't do import.meta.env && import.meta.env.DEV because that breaks tree-shaking
-	import.meta.env = {};
+const node_env = globalThis.process?.env?.NODE_ENV;
+if (!node_env) {
+	throw new Error('conditions should include development or production. Set --conditions in Node.js or consult the documention for your bundler or runtime.');
```

Comment:
```suggestion
	throw new Error('conditions should include development or production. Set --conditions in Node.js or consult the documentation for your bundler or runtime.');
```

Reply:
good catch. updated this with a link and removed that typo in the process